_data/plugins.json

```diff
@@ -119,11 +134,6 @@
     "description": "Wait for a QUnit assertion",
     "date": "2021-11-29T21:09:31.345Z"
   },
-  {
-    "name": "qunitx",
```

Comment:
Not sure why this one dissapeared?

Reply:
Because https://github.com/izelnakri/qunitx/commit/ae698eabd09a25f38519b17dd43eeb2f80d4d90f removed the `qunit-plugin` keyword.